app/helpers/application_helper.rb

```diff
@@ -179,8 +191,18 @@ def holding_location(holding)
   # @return [String] the markup
   def holding_block_search(document)
     block = ''
-    links = search_links(document['electronic_access_1display'])
-    holdings_hash = JSON.parse(document['holdings_1display'] || '{}')
+    portfolio_links = electronic_portfolio_links(document)
+    # Create stub holdings for Alma's electronic profiles so the below logic will work.
+    # Alma does not have an 'Online' library for holdings, so we fake it to keep
+    # the old logic around.
+    # @TODO: Remove when Voyager is gone.
```

Comment:
Which part should be removed when voyager is gone?

Reply:
I expanded the comment.